### PR TITLE
Command/Response directory refactor

### LINKED ISSUE #5: [Refactor] Add directories to organize commands/responses like endpoint paths
For example, `/chains/<chain_id>/blocks` command would be in the `rpc/src/commands/chains/` directory

### PR DESCRIPTION
**Pull request by liam0215**
Expand directory structure so that dir paths match endpoint paths. Right now only added `rpc/src/commands/chains/blocks` and `rpc/src/responses/chains/blocks` directory and moved corresponding commands/responses. Is this too convoluted?

### CODE REVIEW

**Review by LazaroHurtado — APPROVED** [review]
awesome additions! should make things much nicer as we add more commands

File: rpc/src/commands/mod.rs [hunk 1]
```diff
@@ -1,10 +1,9 @@
 use crate::errors::ParseError;
 use crate::responses::Response;
 
-mod get_balance;
-mod get_blocks_in_chain;
-pub use get_balance::GetBalance;
-pub use get_blocks_in_chain::GetBlocksInChain;
+pub mod chains;
+pub use chains::blocks::get_balance::GetBalance;
+pub use chains::blocks::get_blocks_in_chain::GetBlocksInChain;
```
Comment (astherath) [inline rpc/src/commands/mod.rs]: Although the convenience and clarity that being able to call something like `commands::GetBlocksInChain` brings, I think this is really really ugly. 

We should either rework our exports, set a standard as to _what_ gets exported to top level, or fix some of these module hierarchies, so we don't ever have to deal with this type of thing.

If exporting to top level like this is a common pattern in other rust libs, I wouldn't be against keeping it, but it feels like this list of command exports would grow to be obnoxious as the commands grow.

Comment (liam0215) [reply rpc/src/commands/mod.rs]: I would be for removing the top level export. @LazaroHurtado What do you think

Comment (LazaroHurtado) [reply rpc/src/commands/mod.rs]: Yeah Felipe brought up a good point and I agree

**Review by astherath — APPROVED** [review]
lgtm

**Pull request merged**